packages/web-components/src/components/content-block-simple/__stories__/content-block-simple.stories.ts

```diff
@@ -93,11 +93,11 @@ export const Default = (args) => {
               cta-type="${ifDefined(ctaType)}"
               href="${ifDefined(href)}">
               <c4d-card-link-heading>${ctaCopy}</c4d-card-link-heading>
-              <c4d-card-cta-footer>
+              <c4d-card-footer>
                 ${ctaType === 'local' ? ArrowRight20({ slot: 'icon' }) : ''}
```

Comment:
wondering if we can also clean up instances like this... we no longer need to add `ArrowRight20({ slot: 'icon' })` or other icons, we can just use the `cta-type` prop `<c4d-card-footer cta-type="local">` or in this instance `<c4d-card-footer cta-type="${ctaType}">` which will handle it for us and remove an extra import in the story file

Reply:
Actually, you're right. But it doesn't seem like `type` even needs to be added since it's already in the parent (L93)?